modules/roles/README.md

```diff
@@ -6,9 +6,7 @@ Terraform module to create IAM roles
 <!-- BEGIN TFDOCS -->
 ## Requirements
 
-| Name | Version |
-|------|---------|
-| terraform | >= 0.12 |
+No requirements.
```

Comment:
Why was this removed?

Reply:
Just an automatic tfdocs thing, because the terraform version block was deleted. Seemed like just cruft and clutter, at this point.